### ISSUE TITLE
Required two barcodes set as default?

### ISSUE THREAD

**Issue by timacwalker** [rrwick/Porechop#34]
Hi there,

After running porechop with default settings and --verbosity 3 I noticed that it seems a lot of my reads aren't being classified because there isn't a threshold match at both ends. As I understand it, in the default settings this shouldn't be happening. Have I understood the info below correctly? If I have, is there a way to stop this?

Thanks,

Tim.

-------------
...ACGGCTCATCGGGACTAAAAGATCACCTGCATCCACGAGTGACATGACGTCGTCTATACGGGCTCAGCAACGCCCATCCGTTTGGAGCCTCATCAGGTAATTCTTTACTCCCTCAGGATGCAAGCCTATATATAAGGTTTGCGGTGCGGG
  Barcodes:
    start barcodes:     BC02 (78.6%), BC01 (48.0%)
    end barcodes:       BC02 (64.0%), BC01 (59.3%)
    best start barcode: BC02 (78.6%)
    best end barcode:   BC02 (64.0%)
    barcode call:       none
-----------------------

**Comment by rrwick** [comment]
Hi Tim,

You've interpreted this correctly, so it's not immediately obvious to me what's going on.

One thought jumps to mind: are you running Porechop on an Albacore output directory? In the last version I added logic such that if Porechop and Albacore disagree on the barcode, then Porechop won't bin the read. So perhaps your example read didn't get an Albacore bin (wouldn't surprise me if the best hit was only 79%) and so Porechop didn't bin it either.

To make this more clear, I've just added the Albacore core bin to the verbosity level 3 output in Porechop's development branch. You're welcome to try that if you want.

If you're not running Porechop on an Albacore directory, then I don't know what the problem is. Let me know and I can dig deeper.

Ryan
